src/compiler/sys/tests/stencil-sys.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('stencil system', () => {
     expect(newStat.isFile).toBe(true);
 
     const content = await sys.readFile('/dir/file-new');
-    expect('content').toBe('content');
```

Comment:
😬

Reply:
yeah I think I audibly said "ouch" when I saw that 😬

Reply:
Some floating point numbers too, just to be safe